internal/integration/helpers.go

```diff
@@ -207,18 +207,19 @@ func NewTestServer(tb testing.TB) (*serverenv.ServerEnv, *Client) {
 	}
 	mux.Handle("/key-rotation/", http.StripPrefix("/key-rotation", rotationServer.Routes(ctx)))
 
-	// Publish
-	publishConfig := &publish.Config{
-		RevisionToken:            revConfig,
-		MaxKeysOnPublish:         15,
-		MaxSameStartIntervalKeys: 2,
-		MaxIntervalAge:           360 * time.Hour,
-		CreatedAtTruncateWindow:  1 * time.Second,
-		ReleaseSameDayKeys:       true,
-		RevisionKeyCacheDuration: time.Second,
-	}
-
-	publishHandler, err := publish.NewHandler(ctx, publishConfig, env)
+	// Parse the config to load default values.
+	publishConfig := publish.Config{}
+	envconfig.ProcessWith(ctx, &publishConfig, envconfig.OsLookuper())
+	// Make overrides.
+	publishConfig.RevisionToken = revConfig
+	publishConfig.MaxKeysOnPublish = 15
+	publishConfig.MaxSameStartIntervalKeys = 2
+	publishConfig.MaxIntervalAge = 360 * time.Hour
+	publishConfig.CreatedAtTruncateWindow = 1 * time.Second
+	publishConfig.ReleaseSameDayKeys = true
+	publishConfig.RevisionKeyCacheDuration = time.Second
```

Comment:
```suggestion
	publishConfig.RevisionKeyCacheDuration = time.Second
```

above you did `1 * time.Second` - let's be consistent with one or the other?